test/evaluation/submissionFrom.spec.ts

```diff
@@ -67,135 +66,132 @@ describe('Submission requirements tests', () => {
-    const presentationSubmission: PresentationSubmission = evaluationClientWrapper.submissionFrom(pdSchema, vpSimple.verifiableCredential);
-    expect(presentationSubmission.descriptor_map.length).toEqual(2);
-    expect(presentationSubmission.definition_id).toEqual("32f54163-7166-48f1-93d8-ff217bdb0653");
+    evaluationClientWrapper.evaluate(pdSchema, vpSimple);
+    expect(() => evaluationClientWrapper.submissionFrom(pdSchema, vpSimple.verifiableCredential)).toThrowError('Max: expected: 1 actual: 2 at level: 0');
   });
 
   it('Evaluate submission requirements exactly 1 from group B', () => {
     const pdSchema: PresentationDefinition = getFile('./test/resources/sr_rules.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp_general.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp_general.json');
     pdSchema.submission_requirements = [pdSchema.submission_requirements[6]];
     const evaluationClientWrapper: EvaluationClientWrapper = new EvaluationClientWrapper();
     vpSimple.holder = HOLDER_DID;
-    evaluationClientWrapper.evaluate(pdSchema, new VP(vpSimple));
-    const presentationSubmission: PresentationSubmission = evaluationClientWrapper.submissionFrom(pdSchema, vpSimple.verifiableCredential);
-    expect(presentationSubmission.descriptor_map.length).toEqual(2);
-    expect(presentationSubmission.definition_id).toEqual("32f54163-7166-48f1-93d8-ff217bdb0653");
+    evaluationClientWrapper.evaluate(pdSchema, vpSimple);
+    expect(() => evaluationClientWrapper.submissionFrom(pdSchema, vpSimple.verifiableCredential)).toThrowError('Count: expected: 1 actual: 2 at level: 0');
   });
 
   it('Evaluate submission requirements all from group B', () => {
     const pdSchema: PresentationDefinition = getFile('./test/resources/sr_rules.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp_general.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp_general.json');
     pdSchema.submission_requirements = [pdSchema.submission_requirements[7]];
     const evaluationClientWrapper: EvaluationClientWrapper = new EvaluationClientWrapper();
     vpSimple.holder = HOLDER_DID;
-    evaluationClientWrapper.evaluate(pdSchema, new VP(vpSimple));
+    evaluationClientWrapper.evaluate(pdSchema, vpSimple);
     expect(() => evaluationClientWrapper.submissionFrom(pdSchema, vpSimple.verifiableCredential)).toThrowError('Not all input descriptors are members of group B');
   });
 
   it('Evaluate submission requirements all from group A and 2 from group B', () => {
     const pdSchema: PresentationDefinition = getFile('./test/resources/sr_rules.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp_general.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp_general.json');
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.